### ISSUE TITLE
Accessibility: Single product tabs rely on color alone to indicate state

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43617]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The tabs on the single product page use only a simple color change of the text to indicate the hover state. R[elying on color alone is a WCAG violation.](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)

There should be an underline on hover so that there's a visual difference in the state other than color. indicate

### Expected behavior

There will be a visual difference other than color to indicate when the tab link is hovered on.

### Actual behavior

Only color is used to differentiale the hover state. See:
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/product-details/style.scss#L44

### Steps to reproduce

See code above.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by dinhtungdu** [comment]
Adding `needs: design` label as this issue needs input on how we should style the hovered tab.

**Comment by Manussakis** [comment]
I have started looking into this issue.
